src/index.html

```diff
@@ -42,15 +42,29 @@
           </a>
         </div>
         <div class="header__item">
-          <a class="header__lang" href="">EN</a>
+          <a
+            class="header__lang lang-disabled"
+            href=""
```

Comment:
```suggestion
            href="#"
```
не залишай пустими. при натисканні буде перезавантажуватись сторінка